apps/server/src/app.ts

```diff
@@ -65,6 +65,14 @@ app.use('/playback', playbackRouter);
 
 // serve static - css
 app.use('/external', express.static(externalsStartDirectory));
+app.use('/external', (req, res) => {
+  const hasFileEnding = req.url.split('.')[1];
+  if (!hasFileEnding) {
+    res.redirect(`${req.originalUrl}.html`);
+  } else {
+    res.status(404).send(`${req.originalUrl} not found`);
+  }
+});
```

Comment:
can you explain what is the context?
does it not load the `index.html` by default

Reply:
oh you are right, it dose

how about the 404 shold we show an error messege
otherwise it will just fall through to the react page, that could be a bit confusing

Reply:
I dont think we should compromise our app behaviour for this feature.

The old behaviour ensured that as long as you were in the right host, you would see ontime. 
There are several reasons why I feel this is important:
- Users are configuring their own URLs and can get it wrong
- Users can have outdated aliases bookmarked
- users on a phone or tablet can easily make mistakes when typing the URL
- a production can be sharing a link to a less tech savy person who could do with some help

in the cases above, it can be frustrating to keep seeing error pages and we do better in assuring them that they are in the right place

Reply:
btw: just noticed that we copy the contents of the external folder, not the folder itself.
so in production this should be `ontime:4001/demo` in which case we should change the directory to something more meaningful. what do you think?